scripts/base/protocols/conn/thresholds.zeek

```diff
@@ -252,5 +352,19 @@ event conn_packets_threshold_crossed(c: connection, threshold: count, is_orig: b
 		event ConnThreshold::packets_threshold_crossed(c, threshold, is_orig);
 		}
 
-	set_current_threshold(c, F, is_orig);
+	set_current_threshold(c, PACKETS, is_orig);
+	}
+
+event conn_duration_threshold_crossed(c: connection, threshold: interval, is_orig: bool) &priority=5
+	{
+	if ( ! c?$thresholds )
+		return;
+
+	if ( threshold in c$thresholds$duration )
+		{
+		delete c$thresholds$duration[threshold];
+		event ConnThreshold::duration_threshold_crossed(c, threshold, is_orig);
+		}
+
+	set_current_threshold(c, PACKETS, is_orig);
```

Comment:
Should be `DURATION`, not `PACKETS` here?

Reply:
yup, you are obviously right.